app/src/main/java/com/google/samples/apps/sunflower/utilities/Extensions.kt

```diff
@@ -1,39 +0,0 @@
-/*
- * Copyright 2018 Google LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.google.samples.apps.sunflower.utilities
-
-import android.support.v7.app.AppCompatActivity
-import android.view.View
-import androidx.navigation.Navigation
-import androidx.navigation.ui.NavigationUI
-import com.google.samples.apps.sunflower.R
-
-fun AppCompatActivity.setUpAppbar(view: View, isHome: Boolean) {
-    setSupportActionBar(view.findViewById(R.id.toolbar))
-    if (isHome) {
-        NavigationUI.setupActionBarWithNavController(
```

Comment:
I am fine with extension. Just replace setupActionBarWithNavController with normal way to set action bar. I didn't mean to move logic to individual fragment.

Reply:
@sagar-viradiya sry, can't catch ur meaning. Any dummy?